sources/packages/backend/apps/db-migrations/src/migrations/1752517099854-InsertParentDeclarationRequiredParentCanReportNotification.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class InsertParentDeclarationRequiredParentCanReportNotification1752517099854
```

Comment:
What is the benefit of using 2 different migrations here instead of one? IMO one migration is sufficient.